www/.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.env
```

Comment:
Maybe expand this to catch the other kinds of env files we might end up with in a React project. See [here](https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used) for a list of them.

Reply:
Oh cool, I never knew about these. Will add